exporter/src/main/java/io/zeebe/redis/exporter/RedisExporter.java

```diff
@@ -109,34 +115,47 @@ private void connectToRedis() {
     boolean failure = false;
     // try to connect
     try {
+      logger.info("Attempting to establish Redis connection to {}", config.getRemoteAddress().get());
       senderConnection = useProtoBuf ? redisClient.connect(new ProtobufCodec()) : redisClient.connect();
       cleanupConnection = useProtoBuf ? redisClient.connect(new ProtobufCodec()) : redisClient.connect();
-      logger.info("Successfully connected Redis exporter to {}", config.getRemoteAddress().get());
+      logger.info("Successfully connected Redis exporter to {} using {} format", 
+              config.getRemoteAddress().get(), 
+              useProtoBuf ? "protobuf" : "json");
     } catch (RedisConnectionException ex) {
       if (!fullyLoggedStartupException) {
-        logger.error("Failure connecting Redis exporter to " + config.getRemoteAddress().get(), ex);
+        logger.error("Redis connection error: Failed to connect to {}. Error details: {}", 
+                config.getRemoteAddress().get(), ex.getMessage(), ex);
```

Comment:
This change is ok. But needs to be considered in `RedisLateStartupTest` line 72/73 where test waits until the exporter failed to connect to Redis. This ensures that the startup of Redis orchestrated by the test will be after this point.